plugins/obs-filters/noise-suppress-filter.c

```diff
@@ -84,7 +99,18 @@ struct noise_suppress_data {
 	audio_resampler_t *rnn_resampler;
 	audio_resampler_t *rnn_resampler_back;
 #endif
-
+#ifdef LIBNVAFX_ENABLED
+	/* NVAFX handle, one per audio channel */
+	NvAFX_Handle handle[MAX_PREPROC_CHANNELS];
+	uint32_t sample_rate;
+	float intensity_ratio;
+	unsigned int num_samples_per_frame, num_channels;
+	const char *sdk_path;
+	const char *model;
+	/* Resampler */
```

Comment:
Normally, I would have expected there to be a newline here. The code is really squishy.

Reply:
really sorry Jim ! i've added newlines here and elsewhere to make the PR code more readable ...

Reply:
Haha, it's all good, you don't need to apologize! It's just a nitpick more than anything.